### PR TITLE
update machine autoscaler cr

### PR DESCRIPTION
**Pull request by elmiko**
This change updates the guidance for the minimum number machines to
include the functionality for scaling to zero.

### CODE REVIEW

**Comment by elmiko** [comment]
@enxebre ptal, not sure who to contact to push this forward.

**Comment by elmiko** [comment]
this was the only reference to the minimum size that i could find in the docs. should i be looking to create more description about this somewhere?

File: modules/machine-autoscaler-cr.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ which MachineSet this MachineAutoscaler scales, specify or include the name of
 the MachineSet to scale. The MachineSet name takes the following form:
 `<clusterid>-<machineset>-<aws-region-az>`
 <2> Specify the minimum number Machines of the specified type that must remain in the
-specified AWS zone after the ClusterAutoscaler initiates cluster scaling. Do not set this value to `0`.
+specified AWS zone after the ClusterAutoscaler initiates cluster scaling. Do not set this value below `0`.
 <3> Specify the maximum number Machines of the specified type that the ClusterAutoscaler can deploy in the
```
Comment (enxebre) [inline modules/machine-autoscaler-cr.adoc:33]: we should specify "In AWS/GCP/Azure this value can be zero"

**Comment by elmiko** [comment]
updated with @enxebre's suggestion

**Comment by vikram-redhat** [comment]
@kalexand-rh ^

@jianlinliu is this ok?

**Comment by jianlinliu** [comment]
@jhou1 @sunzhaohua2 ^^

**Comment by kalexand-rh** [comment]
@elmiko, is this change valid for 4.5+?

**Comment by elmiko** [comment]
> @elmiko, is this change valid for 4.5+?

yes

**Pull request merged**